--- lit-element.js	2019-09-12 22:40:37.970554100 +0300
+++ lit-element.js.new	2019-09-12 22:40:44.648217600 +0300
@@ -11,12 +11,12 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-import { TemplateResult } from 'lit-html';
-import { render } from 'lit-html/lib/shady-render.js';
+import { TemplateResult } from '../dist/lit-html.js';
+import { render } from '../dist/lit-html.js';
 import { UpdatingElement } from './lib/updating-element.js';
 export * from './lib/updating-element.js';
 export * from './lib/decorators.js';
-export { html, svg, TemplateResult, SVGTemplateResult } from 'lit-html/lit-html.js';
+export { html, svg, TemplateResult, SVGTemplateResult, asyncAppend, asyncReplace, cache, classMap, guard, ifDefined, repeat, styleMap, unsafeHTML, until } from '../dist/lit-html.js';
 import { supportsAdoptingStyleSheets } from './lib/css-tag.js';
 export * from './lib/css-tag.js';
 // IMPORTANT: do not change the property name or the assignment expression.
