--- lit-element.js	2020-06-01 12:11:21.321167966 +0000
+++ lit-element.js.new	2020-06-01 12:10:35.121015469 +0000
@@ -11,11 +11,11 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-import { render } from 'lit-html/lib/shady-render.js';
+import { render } from '../dist/lit-html.js';
 import { UpdatingElement } from './lib/updating-element.js';
 export * from './lib/updating-element.js';
 export * from './lib/decorators.js';
-export { html, svg, TemplateResult, SVGTemplateResult } from 'lit-html/lit-html.js';
+export { html, svg, DefaultTemplateProcessor, directive, isDirective, removeNodes, reparentNodes, noChange, nothing, AttributeCommitter, AttributePart, BooleanAttributePart, EventPart, isIterable, isPrimitive, NodePart, PropertyCommitter, PropertyPart, parts, render, templateCaches, TemplateInstance, SVGTemplateResult, TemplateResult, createMarker, isTemplatePartActive, Template, asyncAppend, asyncReplace, cache, classMap, guard, ifDefined, repeat, styleMap, unsafeHTML, until } from '../dist/lit-html.js';
 import { supportsAdoptingStyleSheets } from './lib/css-tag.js';
 export * from './lib/css-tag.js';
 // IMPORTANT: do not change the property name or the assignment expression.
@@ -197,4 +197,4 @@
  * @nocollapse
  */
 LitElement.render = render;
-//# sourceMappingURL=lit-element.js.map
\ No newline at end of file
+//# sourceMappingURL=lit-element.js.map
